Ambit/Source/AWSUE4Module/Private/S3UEClient.spec.cpp

```diff
@@ -95,7 +96,7 @@ void S3UEClientSpec::Define()
             CreateRequest_2.SetCreateBucketConfiguration(BucketConfig_2);
 
             Aws::S3::Model::CreateBucketOutcome CreateOutcome_2 =
```

Comment:
I checked our ReSharper voting doc and see that we decided to increase our line limit to 120 chars. However, the statement that starts at line 98 is only 105 chars long but is getting hard-wrapped. Why is this?

Reply:
So the ReSharper setting here is a bit finicky. I think it doesn't revert lines that have previously been wrapped, but only wraps lines that are above the threshold. 
This also applies to your other comments. 

There may be a setting to unwrap, but we should be very mindful of trying to utilize that for mass formatting.

Reply:
I'm going to re-run this again with the line break setting allowing for unwrapping for this CR. This should clear up most of the confusion with the existing ones.